### PR TITLE
Cosmin s/tema8

### PR DESCRIPTION
**Pull request by Cosmin0589**

### CODE REVIEW

File: src/test/java/cosminsechel/tests/tema8/TestPage/BaseTest.java [hunk 1]
```diff
@@ -0,0 +1,3 @@
+package cosminsechel.tests.tema8.TestPage;
```
Comment (95teo) [inline src/test/java/cosminsechel/tests/tema8/TestPage/BaseTest.java:1]: Please rename the name of the package from TestPage -> testpage (lowercase for package names as per Java conventions). Thanks!

File: src/test/java/cosminsechel/tests/tema8/TestPage/BaseTest.java [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+
```
Comment (95teo) [inline src/test/java/cosminsechel/tests/tema8/TestPage/BaseTest.java:2]: Add implementation for BaseTest. Here you should have before and after methods for setting up drivers for mobile/desktop, closing browsers.

File: src/test/java/cosminsechel/tests/tema8/TestPage/TestPages.java [hunk 3]
```diff
@@ -0,0 +1,3 @@
+package cosminsechel.tests.tema8.TestPage;
+
```
Comment (95teo) [inline src/test/java/cosminsechel/tests/tema8/TestPage/TestPages.java:2]: Add implementation for tests using Page Objects created in pages package.

**Review by 95teo — CHANGES_REQUESTED** [review]

**Review by tincageorgianaalina — APPROVED** [review]

File: src/test/java/cosminsechel/tests/tema8/testpage/TestPages.java [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    }
+
+    @Test(groups = {"desktop", "mobile"})
+    public void javascriptRedirectPage() {
+        JavaScriptRedirectPage javascriptRedirectPage = new JavaScriptRedirectPage(driver);
+        javascriptRedirectPage.goToPage();
+        javascriptRedirectPage.clickRedirectButton();
+        javascriptRedirectPage.waitForRedirect();
+        assertEquals("Redirected Landing Page", driver.getTitle());
+        driver.navigate().back();
```
Comment (95teo) [inline src/test/java/cosminsechel/tests/tema8/testpage/TestPages.java:28]: No need to navigate back. You can remove this line or add an assert to check after it to check that the url has changed. Thanks!

File: src/test/java/ioananadroflorea/tests/Tema7/BaseTest.java [hunk 5]
```diff
@@ -1,6 +1,6 @@
 package ioananadroflorea.tests.Tema7;
 
-import ioanaandroflorea.driver.BrowserManager;
```
Comment (95teo) [inline src/test/java/ioananadroflorea/tests/Tema7/BaseTest.java:3]: Change ioanaandroflorea to ioanandroflorea and remove the import for your package. Thanks!

File: src/test/java/ioananadroflorea/tests/Tema7/BaseTest.java [hunk 6]
```diff
@@ -11,7 +11,7 @@ public class BaseTest {
 
     @BeforeMethod(groups = "redirect")
     public void initChromeBrowser(){
-        driver = BrowserManager.createChromeDriver();
+        driver = BrowserManager.createDriver();
```
Comment (95teo) [inline src/test/java/ioananadroflorea/tests/Tema7/BaseTest.java:14]: Revert this change. Thanks!

**Review by 95teo — CHANGES_REQUESTED** [review]
You're still missing the .xml files in resources for mobile and desktop groups/suites of tests in order to run tests for mobile/desktop separately.

**Review by 95teo — APPROVED** [review]

**Pull request merged**